crates/starknet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -184,11 +184,9 @@ impl JsonRpcHandler {
         match self.api.starknet.read().await.get_class(block_id.into(), class_hash) {
             Ok(contract_class) => Ok(contract_class.try_into()?),
             Err(Error::NoBlock) => Err(ApiError::BlockNotFound),
-            Err(
-                Error::ContractNotFound
```

Comment:
What happened with ContractNotFound? Both in the match part (left of the arrow) and in the return part (right of the arrow)?

Reply:
According to the spec the https://github.com/starkware-libs/starknet-specs/blob/6fadca1f1abf48523c0aa799a10061f40f973468/api/starknet_api_openrpc.json#L342 the returned errors are BlockNotFound and ClassHashNotFound. get_class method doesnt return ContractNotFound